filter_info.go

```diff
@@ -42,39 +37,48 @@ const (
 
 )
 
-var ErrFieldExist = errors.New("Field is already registered")
+type Context struct {
+	Id               fieldId

```

Comment:
Originally it was not good idea to use separate field for "ID". Can use function `len` to get new ID.
What do you think?